01-node-tutorial/10-fs-sync.js

```diff
@@ -1,12 +1,30 @@
-const { readFileSync, writeFileSync } = require('fs')
+const { writeFileSync, readFileSync } = require('fs');
+
+  // Write 3 lines to a file with append flag
+  writeFileSync('./answers/temporary/fileA.txt', 'Line 1\n', { flag: 'w' });  // 'w' to overwrite or create new file
+  writeFileSync('./answers/temporary/fileA.txt', 'Line 2\n', { flag: 'a' });  // 'a' to append
+  writeFileSync('./answers/temporary/fileA.txt', 'Line 3\n', { flag: 'a' });  // 'a' to append
+
```

Comment:
This is fine, as long as you understand that you could open two files, assign them to variables and write them to a third file that concatenates the two files into the output file.